Automattic/jetpack
# Social: Add support to flag unsupported connections

This change adds support for flagging connections as not supported.

## Proposed changes:

* Added error code in a WPCOM change, which is received here and processed to display the notice if needed.
* Previously broken connections got the 'broken' status code.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?


No.
## Testing instructions:






* Test that the broken connection logic works as before. You can make a connection broken by revoking the access from the social media site.
* For the classic editor, you have to run 'jetpack build packages/publicize' to see the changes
* You can hardcode flag a connection as unsupported, by returning 1 as 'error_code' in 'projects/packages/publicize/src/class-publicize-base.php' for a specific connection. In that case the unsupported notice should appear
